projects/joltify-rwa/index.js

```diff
@@ -6,8 +6,18 @@ const tvl = async (api) => {
     queryV1Beta1({ chain, url: `spv/list_pools` }),
   ]);
 
+  const [price_info] = await Promise.all([
+    queryV1Beta1({ chain, url: `third_party/pricefeed/v1beta1/prices` }),
+  ]);
+
   pools.pools_info.forEach(async pool => {
+    const [market, borrowed_denom] = pool.borrowed_amount.denom.split('-');
+    const market_id = `${market}:usd`;
+    const price = price_info.prices.find(price => price.market_id === market_id)?.price;
+    api.add(borrowed_denom, pool.borrowed_amount.amount * price);
     api.add(pool.usable_amount.denom, pool.usable_amount.amount);
+    api.add(pool.escrow_principal_amount.denom, pool.escrow_principal_amount.amount);
```

Comment:
what is the difference between usable_amount and escrow_principal_amount?

Reply:
The escrow_principal_amount.amount indicates the principal repaid by the SPV and locked into the pool so that the investors can claim their investing assets from the pool once the due time reached.

Reply:
Similar to escrow_interest_amount, which indicates the repaid interest by the SPV.

Reply:
The usable amount is the asset amount invested or put in the pool by users/investors for the SPV to borrow.
Basically, the escrow_principal and escrow_interest in the pool are from the SPV/borrower, the usable amount in the pool are from the users/investors.